### PR TITLE
Fix crash using `Color.css` with rgb(), rgba().

### PR DESCRIPTION
**Pull request by daohoangson**
I have added 2 test cases related to this.

Related to dart-lang/tools#1167

### CODE REVIEW

**Comment by kevmoo** [comment]
please rebase if you'd like to land this!

**Comment by daohoangson** [comment]
I have resolved the conflicts and made sure CI passes. Please review and squash it on merge. Thank you.

File: test/color_test.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
+library color_test;
```
Comment (kevmoo) [inline test/color_test.dart]: please remove the library directive and include the standard Dart copyright header.

Thanks!

**Review by kevmoo — CHANGES_REQUESTED** [review]

File: test/color_test.dart (hunk 1 above)
Comment (daohoangson) [reply test/color_test.dart]: I'll include the copyright header however please confirm the removal of the library directive. Other files have similar header:

https://github.com/dart-lang/csslib/blob/11d4cf4c292c82245ec40a45671fc4e8334336c7/test/declaration_test.dart#L5

https://github.com/dart-lang/csslib/blob/11d4cf4c292c82245ec40a45671fc4e8334336c7/test/error_test.dart#L5

Comment (natebosch) [reply test/color_test.dart]: Some older libraries still have them but we avoid adding them in new files. I've gone ahead and pushed the cleanup for that bit for expediency. Sorry for the delays.

**Comment by natebosch** [comment]
I took a closer look, I think we want to keep these as `int` earlier, but we can use `toInt()` instead of `as it`.

https://github.com/dart-lang/csslib/pull/104/commits/c7467fed1a803ac3c2594419fb54f1edd344969a

cc @jakemac53 for a second googler review since I've proposed a different approach. WDYT?

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**